src/components/Fund/FundPoints.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function FundPoints({ description }) {
+  return (
+    <div style={{ display:"flex", flexDirection:"column", alignItems:"flex-start"}}>
```

Comment:
I have added an another file named circle.svg, we should import that here.

Reply:
done